metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bro/BasicBroParser.java

```diff
@@ -34,6 +35,7 @@
 
   protected static final Logger _LOG = LoggerFactory
           .getLogger(BasicBroParser.class);
+  public static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("0.0#####");
```

Comment:
DecimalFormat is not threadsafe, can you please make this a `ThreadLocal<DecimalFormat>`?